consensus/bor/bor.go

```diff
@@ -888,9 +888,36 @@ func (c *Bor) Finalize(chain consensus.ChainHeaderReader, header *types.Header,
 		err           error
 	)
 
+	// Set state sync data to blockchain
+	bc := chain.(*core.BlockChain)
+	bc.SetStateSync(stateSyncData)
+	tracer := bc.GetTracingHooks()
```

Comment:
Instead, simply here
```suggestion
	tracer := chain.(*core.BlockChain).GetTracingHooks()
```